rocketpy/motors/TankGeometry.py

```diff
@@ -164,6 +164,78 @@ def inverse_volume(self):
             lambda v: v / (np.pi * self.average_radius**2),
         )
 
+    @cache
+    def balance(self, lower, upper):
+        """
+        The volume balance of the tank as a function of height.
+
+        Returns
+        -------
+        Function
+            Tank centroid as a function of height.
+        """
+        height = self.area.identityFunction()
+        balance = (height * self.area).integralFunction(lower, upper)
+
+        # Correct naming
+        balance.setInputs("height (m)")
+        balance.setOutputs("balance (m⁴)")
+
+        return balance
+
```

Comment:
This is a little hard for me to understand. What is `balance`?

Reply:
I don't like this name too. Feel free to suggest a better one. To sum up I had to come with a name for this integral:

$$\int_{h_i}^{h_f} h \cdot A(h) \thinspace dh$$

I baptized it `volume balance` since it is the "mass balance" of the tank before the multiplication by the density. But I feel that this name is not descriptive enough.

Reply:
Okay, this is a tough one. I'd suggest calling the variable `volume_balance` then and explaining a bit better what it represents in the docstring

Reply:
See [18acbac](https://github.com/RocketPy-Team/RocketPy/pull/354/commits/18acbac1cded5c04ceaf3d28c8906596cf50e14c). I will mark this as resolved since there is a more up to date comment on the same issue.